drivers/char/vhm/vhm_dev.c

```diff
@@ -157,10 +157,15 @@ static ssize_t vhm_dev_write(struct file *filep, const char *buffer,
 static void update_assigned_vf_state(uint16_t bdf, bool is_assigned)
 {
 	struct pci_dev *dev = NULL;
+	struct pci_bus *bus;
 
-	dev = pci_get_slot(pci_find_bus(0, PCI_BUS_NUM(bdf)),
-			(bdf & 0xFF));
+	bus = pci_find_bus(0, PCI_BUS_NUM(bdf));
+	if (!bus) {
+		pr_err("vhm: invalid bus number %d\n", PCI_BUS_NUM(bdf));
```

Comment:
Is this a really error?  Maybe the error log can be suppressed and ignored.

Reply:
Sure, I think it is a sanity check, I will remove the error log, thanks for your suggestion